ui/plugin-system/src/runtime/TimeRangeProvider/TimeRangeProvider.tsx

```diff
@@ -19,7 +19,12 @@ import {
   TimeRangeValue,
   toAbsoluteTimeRange,
 } from '@perses-dev/core';
-import { useSetRefreshIntervalParams, useSetTimeRangeParams } from './query-params';
+import {
+  useSetRefreshIntervalParamsWithQueryParams,
+  useSetRefreshIntervalParams,
+  useSetTimeRangeParamsWithQueryParam,
+  useSetTimeRangeParams,
+} from './query-params';
```

Comment:
Maybe the ones that don´t use query params should not live in a file named query-params